lib/github/tree.js

```diff
@@ -29,13 +29,29 @@ export default class GitHubTree {
 
   async _getLastCommit() {
     const { request, owner, repo, branch, path } = this;
+    console.log('asdasd', COMMIT_QUERY, {
+      owner,
+      repo,
+      branch,
+      path
+    });
     const data = await request.gql(COMMIT_QUERY, {
       owner,
       repo,
       branch,
       path
     });
-    return data.repository.ref.target.history.nodes[0].oid;
+
+    console.log('data', data);
+    const targetHistoryNodes = data.repository.ref.target.history.nodes;
+    if (!targetHistoryNodes.length) {
+      this.cli.stopSpinner(
+        `Cannot find WPT for "${path}". Please check the WPT name.`,
+        this.cli.SPINNER_STATUS.FAILED
+      );
+      process.exit();
```

Comment:
We need to set a non-zero exit code so the terminal (or whatever has spawn the process) can report the error
```suggestion
      process.exit(1);
```
But IMO it would be more user-friendly to throw an actual error, with a stack trace, so anyone trying to debug the code can follow what's going wrong.
```suggestion
      throw new Error('Cannot find last commit');
```

Reply:
SGTM

Reply:
Not sure throw an error is better than `process.exit(1)` 😄 As `this.cli.stopSpinner` already has a clear `message` to indicate the cause and easy to locate in the code base and `this.cli.SPINNER_STATUS.FAILED` to indicate something went wrong + error exit code.